packages/ag-charts-enterprise/src/error-bar/errorBar.ts

```diff
@@ -126,8 +139,10 @@ export class ErrorBars extends _ModuleSupport.BaseModuleInstance implements _Mod
         const { nodeData } = this;
         const points = nodeData[index];
         if (points) {
-            node.points = points;
-            node.updatePath();
+            const defaults: ErrorBarNodeProperties = ERROR_BARS_THEME.errorBar;
```

Comment:
Note: this a temporary workaround. Getting themes to work correctly for error bars will probably making changes to `packages/ag-charts-community/src/chart/factory/setupModules.ts` to support `'series-option'` type modules. I will do this in a follow up PR